modules/git/commit.go

```diff
@@ -248,6 +248,20 @@ func CommitChanges(repoPath string, opts CommitChangesOptions) error {
 	return err
 }
 
+func allCommitsCount(repoPath string) (int64, error) {
+	stdout, err := NewCommand("rev-list", "--all", "--count").RunInDir(repoPath)
+	if err != nil {
+		return 0, err
+	}
+
+	return strconv.ParseInt(strings.TrimSpace(stdout), 10, 64)
+}
+
+// AllCommitsCount returns count of all commits in repository
+func AllCommitsCount(repoPath string) (int64, error) {
+	return allCommitsCount(repoPath)
```

Comment:
Why the indirection here? I could see it being reasonable to add a function on git.Repository that basically just calls a non-localised repo only function but this seems unnecessary.